<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="cNameOperationsLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateAuthenticatedButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addAuthenticatedButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnAuthenticatedButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Go back...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 60px;
</string>
   </property>
   <property name="text">
    <string>ELISTA</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>-10</y>
     <width>21</width>
     <height>621</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
</string>
   </property>
   <property name="text">
    <string>Hello,</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>1121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>To-do lists, personified.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 50px;</string>
   </property>
   <property name="text">
    <string>Task Adder</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="taskNameField">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
</string>
   </property>
   <property name="text">
    <string>Task name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
</string>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="addTaskTypeBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Personal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Academic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Miscellaneous</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
</string>
   </property>
   <property name="text">
    <string>Priority:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="addTaskPriorityBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Urgent</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>High</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Average</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Low</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Optional</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="addTaskDeadlineBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
</string>
   </property>
   <property name="text">
    <string>Deadline:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTaskSubmitBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>497</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10px;
</string>
   </property>
   <property name="text">
    <string>DD/MM/YYYY</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
